app/src/main/java/org/simple/clinic/summary/medicalhistory/MedicalHistorySummaryUiRenderer.kt

```diff
@@ -0,0 +1,37 @@
+package org.simple.clinic.summary.medicalhistory
+
+import org.simple.clinic.medicalhistory.MedicalHistory
+import org.simple.clinic.mobius.ViewRenderer
+import org.simple.clinic.util.ValueChangedCallback
+
+class MedicalHistorySummaryUiRenderer(
+    private val ui: MedicalHistorySummaryUi
+) : ViewRenderer<MedicalHistorySummaryModel> {
+
+  // Added to keep regression tests passing, otherwise whenever the
+  // model updates, the render method will call the UI methods again
+  // which will cause tests to break.
```

Comment:
I don't think we need this as a comment here. We can instead update the `migration-to-mobius` doc to add this use case. And maybe move this comment to the commit instead?

Reply:
Good point, I'll make this change.